content/en/legacy/flux/references/garbagecollection.md

```diff
@@ -5,18 +5,18 @@ weight: 50
 ---
 
 Part of syncing a cluster with a git repository is getting rid of
-resources in the cluster that have been removed in the repository. You
+resources in the cluster that has been removed in the repository. You
 can tell `fluxd` to do this "garbage collection" using the command-line
 flag `--sync-garbage-collection`. It's important to know how it
-operates, and appreciate its limitations, before enabling it.
+operates and appreciate its limitations before enabling it.
```

Comment:
Can you explain what you are trying to do here?

Reply:
I have grammarly enabled in my IDE, and it just made this recommendation.

Reply:
I'll let an English native-speaker review this.

Reply:
@kingdonb have you got a chance to review this PR?